parsl/tests/test_cleanup/test_cleanup.py

```diff
@@ -0,0 +1,34 @@
+
+from parsl import *
+from parsl.configs.local import localThreads as config
+dfk = DataFlowKernel(config=config)
+
+
+@App('python', dfk)
+def slow_double(x, dur=0.1):
+    import time
+    time.sleep(dur)
+    return x * 5
+
+
+def test_cleanup_behavior_221():
+    """ A1 A2 A3   -> cleanup
+        B1 B2 B3
+
+    """
+    round_1 = []
+    for i in range(0, 10):
+        f = slow_double(i)
+        round_1.append(f)
+
+    round_2 = []
+    for i in round_1:
+        f = slow_double(i)
+        round_2.append(f)
+
+    dfk.cleanup()
+
```

Comment:
I think we need to assert something here?

Reply:
Well, I don't know what to assert yet. I think the mistake was in this test getting added to this PR.
This was for figuring out the #221 behavior.

Reply:
This (incomplete) test should've gone in on a separate PR. I was watching for cleanup behavior, and there isn't something to assert on here yet.